healthvault-ios-sdk/Classes/Controllers/MHVBrowserAuthBroker.m

```diff
@@ -20,13 +20,14 @@
 #import "NSError+MHVError.h"
 #import "MHVValidator.h"
 
-typedef void (^MHVSignInCompletion)(NSURL *_Nullable successUrl, NSError *_Nullable error);
-
 @interface MHVBrowserAuthBroker ()
 
 @property (nonatomic, strong) MHVSignInCompletion completion;
 @property (nonatomic, strong) dispatch_queue_t authQueue;
 @property (nonatomic, strong) NSURL *endUrl;
+@property (nonatomic, strong) NSURL *startUrl;
+
+@property (nonatomic, strong) MHVBrowserController *browserController;
```

Comment:
We should probably move this out of the Controllers directory and put it in the Authentication
 directory.